grid.cpp

```diff
@@ -241,11 +241,11 @@ void initializeGrids(
             balanceLoad(mpiGrid, sysBoundaries);
          }
       } else if (P::refineOnRestart) {
+         // Get good load balancing for refinement
```

Comment:
Add a "deprecated" comment here as well. I would also like to see a comment above on  line 235 about what Force refinement does, and what that i parameter to the call actually is.